ui/src/app/workflows/components/workflows-list/workflows-list.tsx

```diff
@@ -29,6 +29,8 @@ import {WorkflowsSummaryContainer} from '../workflows-summary-container/workflow
 import {WorkflowsToolbar} from '../workflows-toolbar/workflows-toolbar';
 
 import './workflows-list.scss';
+import useTimestamp, {TIMESTAMP_KEYS} from '../../../shared/use-timestamp';
+import {TimestampSwitch} from '../../../shared/components/timestamp';
```

Comment:
same as above

Reply:
This was partly individually fixed separately in https://github.com/argoproj/argo-workflows/pull/13160, and #13813 fixed the remaining grouping / alphabetization issue as per [above](https://github.com/argoproj/argo-workflows/pull/13284#discussion_r1792540976)